docs/proposals/scheduling/average-replicas-assign/average-replicas-assign.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Karmada distributes the replicas evenly based on spread constraint
```

Comment:
can you re-summarize the title? in fact, we did not consider spread constraints during the design.